src/components/ArticlePage/index.jsx

```diff
@@ -0,0 +1,107 @@
+import VerticalMargin from '../../components/VerticalMargin';
+import HorizontalMargin from '../../components/HorizontalMargin';
+import RatingStars from '../../components/RatingStars';
+import RatingStarsBox from '../../components/RatingStarsBox';
+import HorizontalLine from '../../components/HorizontalLine';
+import TextAreaInput from '../../components/TextAreaInput';
+import CommentEntries from '../../components/CommentEntries';
+import { getOneArticle } from '../../redux/actions/articles';
+import './style.scss';
+
+
+export class SingleArticleView extends Component {
+
+  componentDidMount() {
+    const { fetchArticle, match } = this.props;
+    let articleId = match.params.articleId;
+    fetchArticle(articleId);
+  }
+
+  render() {
+    const { match, articles = [] } = this.props;
+    const { articleId } = match.params;
+    const article = articles.find(item => item.id === articleId) || {};
+
+    const { 
+      title, 
+      featuredImage = '/assets/img/map-typing.jpg', 
+      author: { username = '', imageUrl: userAvatarImage = '' } = {}, 
+      body,
+      comments = []
+    } = article;
+
+  
+    return (
+      <div>
+        <div className='banner-section'>
+          <div className='banner-image'>
+            <img src={featuredImage} alt='banner background' />
+          </div>
+          <div className='transparent-overlay' />
+          <div className='banner-content'>
+            <div className='auto-container'>
+              <div className='banner-content-title'>{title}</div>
+              <div className='avatar-text-group'>
+                <div className='user-profile-picture'>
+                  <RoundedImage imageSource={userAvatarImage} alt='profile picture'/>
+                  <VerticalMargin className='hide-for-medium' size={50} />
+                </div>
+                <HorizontalMargin className='show-for-medium' size={10} />
+                <div className='article-username-rate-group'>
+                  <div className='article-header-username'>{username}</div>
+                  <div className='article-header-rating'>
+                    <RatingStars rateNumber={4} />
+                  </div>
+                </div>
+              </div>
+            </div>
+          </div>
+        </div>
+        <div className='body-section'>
+          <div className='auto-container wide-padding'>
+            <div className='article-body'>
+              <div dangerouslySetInnerHTML={{ __html: body }} />
+            </div>
+            <HorizontalLine />
+            <div className='section-article-info'>  
+              <div>Rate this Article <RatingStarsBox rateNumber={4} /></div>
+              <VerticalMargin className='show-for-medium' size={10} />
+              <div>
+                <div className='section-comment-create'>
+                  <div>Leave comments</div>
+                  <TextAreaInput className='article-comment-field' text='Your message' />
```

Comment:
Nice job @jsamchineme. What do you think about using a placeHolder instead of text? Using the text means a user will always have to clear the text input area first before being able to type.

Reply:
Thanks for the mention, peace. I will update it to use a placeholder as well as text. The `placeholder` text is displayed when the user has not provided input, while `text` is the actual input.